custom/icds_reports/utils/aggregation_helpers/monolith/agg_awc.py

```diff
@@ -230,13 +230,14 @@ def updates(self):
                 owner_id,
                 sum(open_count) AS cases_household
             FROM "{household_cases}"
+            WHERE opened_on<= %(end_date)s
```

Comment:
Should I make this same change to citus also?

Reply:
yes please. You'll also need to run `manage.py update_icds_aggregation_output`

Reply:
DOne